### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by mboukhalfa**
Add FakeIPA to the root README file

### CODE REVIEW

**Review by Rozzii — COMMENTED** [review]
/approve

I am fine with this commit but if possible create further PRs that extend the READMI with documentation related to the usage of the fake-ipa tool eg. an example workflow or something similar.

**Comment by tuminoid** [comment]
> /approve
> 
> I am fine with this commit but if possible create further PRs that extend the READMI with documentation related to the usage of the fake-ipa tool eg. an example workflow or something similar.

Can we please make such additions under fake-ipa README. I don't think the global readme needs more than this quick intro, and a link to further reading.

File: README.md [hunk 1]
```diff
@@ -60,3 +60,17 @@ functionality:
...
+
+FakeIPA simulate the IPA by:
+
+- Running an API server with the needed real IPA endpoint.
+- Send back fake inspection data when requested.
+- Lookup the node and save tokens.
+- Heartbeating to Ironic API with several threads looping over
+  a queue of fake agents.
+- Faking the sync/async commands needed by ironic to inspect,
+  clean and provision a node.
```
Comment (tuminoid) [inline README.md:76]: Add a "read more" link to fake-ipa/README.md for more docs.

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**